### PR TITLE
Release automation: Tekton 0.7 support, publish openshift yaml, and pin image coordinates

### PR DESCRIPTION
**Pull request by mnuttall**
# Changes

- Tekton 0.7 support for https://github.com/tektoncd/experimental/issues/262
- Openshift yaml publication and image pinning for https://github.com/tektoncd/experimental/issues/208

See https://console.cloud.google.com/storage/browser/tekton-releases/webhooks-extension/test-issue.262.take1/previous/0.2.1-test7/?project=tekton-releases

Since that release I've changed `release.yaml` to `tekton-webhooks-extension-release.yaml` and adjusted the Readme accordingly. 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: webhooks-extension/config/latest/gcr-tekton-webhooks-extension.yaml [hunk 1]
```diff
@@ -201,19 +201,19 @@ spec:
     image: maiwj/kubernetes-python-client:latest
     env:
       - name: PIPELINERUNS
-        value: ${inputs.params.pipelineruns}
+        value: $(inputs.params.pipelineruns)
```
Comment (a-roberts) [inline webhooks-extension/config/latest/gcr-tekton-webhooks-extension.yaml]: Folder name has changed from release to latest, https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/InstallNightlyBuild.md references `release` and you've not changed it on this PR (I only see the Tekton folder's markdown doc being changed)

File: webhooks-extension/config/latest/openshift-tekton-webhooks-extension.yaml [hunk 2]
```diff
@@ -296,19 +296,19 @@ spec:
     image: maiwj/kubernetes-python-client:latest
```
Comment (a-roberts) [inline webhooks-extension/config/latest/openshift-tekton-webhooks-extension.yaml]: Is this another folder name change? If so please check what's at https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/InstallNightlyBuild.md  and anything else at https://github.com/tektoncd/experimental/tree/master/webhooks-extension/docs

File: webhooks-extension/test/presubmit-tests.sh [hunk 3]
```diff
@@ -76,8 +76,8 @@ function node_test() {
     echo "LATEST HASH: $hash"
 
     yaml=$(grep -i "tekton-dashboard-bundle-location:" config/extension-service.yaml | cut -f 2 -d ':' | cut -f 2 -d '.')
-    yaml2=$(grep -i "tekton-dashboard-bundle-location:" config/release/gcr-tekton-webhooks-extension.yaml | cut -f 2 -d ':' | cut -f 2 -d '.')
-    yaml3=$(grep -i "tekton-dashboard-bundle-location:" config/release/openshift-tekton-webhooks-extension.yaml | cut -f 2 -d ':' | cut -f 2 -d '.')
+    yaml2=$(grep -i "tekton-dashboard-bundle-location:" config/latest/gcr-tekton-webhooks-extension.yaml | cut -f 2 -d ':' | cut -f 2 -d '.')
+    yaml3=$(grep -i "tekton-dashboard-bundle-location:" config/latest/openshift-tekton-webhooks-extension.yaml | cut -f 2 -d ':' | cut -f 2 -d '.')
     echo "YAML HASH in config/extension-service.yaml: $yaml"
     echo "YAML HASH config/release/gcr-tekton-webhooks-extension.yaml: $yaml2"
```
Comment (a-roberts) [inline webhooks-extension/test/presubmit-tests.sh]: These echos should be for `latest` too now?

File: webhooks-extension/tekton/publish.yaml [hunk 4]
```diff
@@ -61,21 +68,41 @@ spec:
...
+    - /bin/sh
+    args:
+    - -ce
+    - |
+      curl https://raw.githubusercontent.com/tektoncd/dashboard/master/tekton/scripts/lockdown.py --output lockdown.py
+      chmod +x lockdown.py
+      pip install docker
+      ./lockdown.py --omit webhooks-extension --path latest/tekton-webhooks-extension-release.yaml
+      ./lockdown.py --omit webhooks-extension --path latest/openshift-tekton-webhooks-extension-release.yaml
+    volumeMounts:
```
Comment (a-roberts) [inline webhooks-extension/tekton/publish.yaml]: Changes look good now, how come we're now doing docker socket mounting?

Comment (mnuttall) [reply webhooks-extension/tekton/publish.yaml]: for the record it's required by the lockdown.py script

**Pull request merged**